manifests/m/Mega/MEGASync/4.12.2/Mega.MEGASync.locale.en-US.yaml

```
@@ -0,0 +1,24 @@
+# Created with Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: Mega.MEGASync
+PackageVersion: 4.12.2

```

```suggestion
PackageVersion: 4.12.2.0
```
Please update file name and path to match this change.